platform.spec.js

```diff
@@ -1,11 +1,12 @@
 describe('suite of tests for the platform',function () {
-  it('Router base js should not be changed', function() {
-    console.log('Unit Test: Router base js should not be changed.');
-    var checksum = objectHash(routerSetupConfig, { algorithm: 'md5', encoding: 'base64' });
-    expect(checksum).toBe('2f10vxKAVe5PHd17c8tJdA==');
+  it('Router base js\' status object should not be changed', function() {
+    console.log('Router base js\' status object should not be changed');
+    var checksum = md5(JSON.stringify(routerSetupConfig.status));
```

Comment:
JSON.stringify() is being run on the status object to convert the object to a string.